RELEASE/scripts/autoscend.ash

```diff
@@ -12927,8 +12962,60 @@ boolean L9_chasmBuild()
 		return true;
 	}
 
+	// -Combat is useless here since NC is triggered by killing Orcs...So we kill orcs better!
+	asdonBuff($effect[Driving Intimidatingly]);
+
+	// Check our Load out to see if spells are the best option for Orc-Thumping
+	skill useSpellsInOrcCamp = $skill[none];
```

Comment:
is this variable actually used? I mean as a skill. seems like its just used essentially as a boolean condition, "do I have a cold spell to overkill with". if so I'd rather it be a boolean, makes the intent more clear.

Reply:
I originally had it a boolean, then in sl_ascend they said this was the better "cleaner" way of doing it.

Reply:
I agree with @macgregor we're not using the spell here so a simple boolean is a better choice.

Reply:
I switched it back to boolean